network-api/networkapi/wagtailpages/pagemodels/libraries/rcc/detail_page.py

```diff
@@ -72,36 +70,30 @@ class RCCDetailPage(BasePage):
 
     content_panels = wagtail_models.Page.content_panels + [
         image_handlers.FieldPanel("cover_image"),
-        # TODO: Reactivate once links are implemented
-        # edit_handlers.InlinePanel("rcc_links", heading="Article links"),
+        edit_handlers.InlinePanel("rcc_links", heading="Article links"),
```

Comment:
Do we need the `rcc_` prefix in these fields?

Reply:
That's a good point. I think this was a bad port from **Research Hub**. The original was `research_links`, so I ended up putting this as `rcc_links`, but maybe a better name would be `article_links` and `article_authors` for the one below. What do you think?

Reply:
> However, now that I'm thinking about the refactoring and generalizing some of the methods, I think it would be better to remove the prefixes from both libraries, i.e. having only links and authors, since the page already has the context into whether this is from RCC or from ResearchHub. This would allow methods such as get_authors and get_author_count to get more general.

100% 👍

Reply:
#10697